### DISCUSSION TITLE
Insert link only inserts [[]] without the title

### DISCUSSION THREAD

**Discussion by lokesh-krishna** [nvim-telekasten/telekasten.nvim#192] [Q&A]
I'm on the latest version of the plugin and on Neovim 0.8.0. Running `Telekasten insert_link` simply inserts `[[]]` instead of the note I've selected. I'm not sure how to go about debugging this and would really appreciate any help in this regard. 

I do have a keybind for it: `inoremap <leader>[ <ESC>:lua require('telekasten').insert_link()<CR>` but I run into the same issue even when directly running the command so I don't think my keybind is the problem?

**Comment by lokesh-krishna** [comment]
Anyone else facing this problem? Really not sure how to go about troubleshooting it.

**Comment by lambtho12** [comment]
Is it related to: https://github.com/renerocksai/telekasten.nvim/issues/190?

I think I (finally) have some time to work on Telekasten a bit in the following days. I will try to crunch a few bugs, this one is definitely in my top priorities.
